src/pages/Register/Register.scss

```
@@ -0,0 +1,69 @@
+@mixin flex($justify: center, $align: center) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+}
+
+.register-header {
+  min-width: 500px;
+
+  .register-nav {
+    @include flex;
+    border-bottom: 1px solid #f0f0f0;
+
+    h1 {
+      padding: 10px 170px;
+      font-size: 17px;
+      font-weight: 500;
+    }
+
+    button {
+      border: 0;
+      background-color: white;
+      font-size: 20px;
+    }
+  }
+}
+
+.register-body {
+  @include flex;
+  min-height: 80vh;
+
+  .content {
+    margin-top: 40px;
+
+    .inputs {
+      @include flex($align: flex-start);
+      flex-direction: column;
+
+      input {
+        margin-bottom: 20px;
+        width: 400px;
+        height: 50px;
+        border-radius: 5px;
+        border: 1px solid #dad6d6;
+        text-indent: 10px;
+      }
+
+      .message {
+        margin-bottom: 10px;
+        margin-left: 5px;
+        font-size: 15px;
+      }
+    }
+
+    button {
+      width: 400px;
+      height: 50px;
+      border-radius: 5px;
+      border: 1px solid white;
+      background-color: #0095f6;
+      color: white;
+    }
+
+    button:disabled {
+      color: rgba(136, 136, 136, 0.5);
+      background-color: rgb(237, 237, 237);
+    }
```

부모 선택자(`&`)를 활용할 수 있습니다.
```suggestion
    button {
      width: 400px;
      height: 50px;
      border-radius: 5px;
      border: 1px solid white;
      background-color: #0095f6;
      color: white;
      
      &:disabled {
        color: rgba(136, 136, 136, 0.5);
        background-color: rgb(237, 237, 237);
      }
    }
```